lib/error.js

```diff
@@ -1,8 +1,8 @@
 
-
 //eslint-disable-next-line max-len
 //@see http://stackoverflow.com/questions/31089801/extending-error-in-javascript-with-es6-syntax
 
+const { curry } = require('ramda')
```

Comment:
You know, I never really thought of that. I talked to Jason and he said there's really no benefit/disadvantage (if not negligible) to doing either. So I will require the entire R for the sake of readability.

Thoughts @scull7 ?